ci/cfg/ci_aes_sca_random_cw305.yaml

```diff
@@ -24,21 +24,22 @@ waverunner:
 capture:
   scope_select: husky
   # scope_select: waverunner
-  num_traces: 500
+  num_traces: 10000
```

Comment:
The reason for using a relative low number of traces was to keep the CI execution time low. Is capturing 10000 traces still fast enough and do we need that many traces for a quick visual inspection whether traces are OK or not?

Reply:
Fair point. In CI, the batch capture rate is around 1500 traces/s so it's fairly quick. However, I've now reduced it 4000, as this seems already sufficient for a specific TVLA (results will not be very meaningful though of course!). I plan to add some CI step for the specific TVLA in a follow-up PR, see other issue. And this step should use the traces captured here.